cpp/CMakeLists.txt

```diff
@@ -50,6 +50,11 @@ option(BUILD_SHARED_LIBS "Build cuDF shared libraries" ON)
 option(JITIFY_USE_CACHE "Use a file cache for JIT compiled kernels" ON)
 option(CUDF_BUILD_TESTUTIL "Whether to build the test utilities contained in libcudf" ON)
 mark_as_advanced(CUDF_BUILD_TESTUTIL)
+# TODO: Should this be on or off by default? Is it an advanced option?)
```

Comment:
I am fine with it being on by default, as an advanced option. That matches the test utilities above.
```suggestion
```